tex2pdf_service/Makefile

```diff
@@ -45,7 +45,7 @@ app.docker:
 	@echo "dockerport: ${app_port}"
 	@echo "tag: ${appl_tag}"
 	@echo "name: ${app_name}"
-	docker build -f ./Appliance.Dockerfile -t ${app_tag} .
+	docker buildx build -f ./Appliance.Dockerfile -t ${app_tag} .
```

Comment:
Not sure whether some devs here have M1/M2 based macbooks, but then passing in `--platform=linux/amd64` ensures that we build amd64 docker images, and not aarch64.

Reply:
This is for local docker only. Not sure it matters.

Reply:
Ok, if someone runs M1/M2 hardware, we can look at it later, thanks